webserver/tyrus/pom.xml

```diff
@@ -58,7 +58,6 @@
         <dependency>
             <groupId>org.glassfish.tyrus</groupId>
             <artifactId>tyrus-client</artifactId>
-            <scope>test</scope>
```

Comment:
@spericas Why the client?

Reply:
I don't think this is going to work without `requires org.glassfish.tyrus.container.grizzly.client;`

Reply:
Please no. Let's talk to Tyrus to get a module that does not explicitly depend on a client